crates/orderbook/src/api/post_solver_competition.rs

```diff
@@ -8,6 +8,18 @@ use shared::api::convert_json_response_with_status;
 use std::{convert::Infallible, sync::Arc};
 use warp::{reply::with_status, Filter, Rejection};
 
+#[async_trait::async_trait]
+pub trait SolvableOrdersCache: Send + Sync {
+    async fn update_next_solver_competition_id(&self) -> anyhow::Result<()>;
+}
```

Comment:
Is the plan to move more methods of the `SolvableOrdersCache` into the trait?
If not I think the name could be more specific to its purpose. Maybe `SolverCompetitionIdUpdating`?

Reply:
There is no plan for more methods. This encapsulates the one method this module needs. I couldn't think of a good name for it. I feel the current name still makes some sense as this represents the SolvableOrdersCache but from the perspective of this module. I don't expect anyone outside of this module to use the trait.